src/window/window_wrapper.rs

```diff
@@ -172,6 +176,35 @@ impl WinitWindowWrapper {
         wrapper
     }
 
+    fn load_image(image: &str) -> Option<Image> {
+        if image.is_empty() {
+            None
+        } else {
+            let skia_data = Data::from_filename(std::path::Path::new(image))?;
```

Comment:
I think we might want to use https://docs.rs/shellexpand/latest/shellexpand/fn.tilde_with_context.html here so that you can easily refer to the home directory